### DISCUSSION TITLE
Just two UI feature Requests

### DISCUSSION THREAD

**Discussion by itscz-org** [FreeTubeApp/FreeTube#6484] [General]
Thank you for the great app. I just have two little feature requests for the UI:

1. Add an option to hide the large play/pause button in the center of the video. I often pause DIY videos or walkthroughs and have no clear view to the still image because of that.
2. Add a simple reload button. When pausing videos a long time or after resuming from hibernation the next day, there are lot of API errors shown when pressing play again. As there is no reload, i need to go back and find the link again i originally came from.

Thanks,
Chris

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
1. duplicate of https://github.com/FreeTubeApp/FreeTube/issues/6149
2. duplicate of #1005
